doc/man/sphinx-build.rst

```diff
@@ -174,56 +198,77 @@ Options
 
    .. versionadded:: 1.6
 
-.. option:: -v
+.. option:: -v, --verbose
 
    Increase verbosity (loglevel).  This option can be given up to three times
    to get more debug logging output.  It implies :option:`-T`.
 
    .. versionadded:: 1.2
 
-.. option:: -q
+.. option:: -q, --quiet
 
    Do not output anything on standard output, only write warnings and errors to
    standard error.
 
-.. option:: -Q
+   .. versionchanged:: 7.3
+      Add ``--quiet`` long option.
+
+.. option:: -Q, --really-quiet
 
    Do not output anything on standard output, also suppress warnings.  Only
    errors are written to standard error.
 
-.. option:: -w file
+   .. versionchanged:: 7.3
+      Add ``--really-quiet`` long option.
+
+.. option:: -w file, --write file
 
    Write warnings (and errors) to the given file, in addition to standard error.
 
    .. versionchanged:: 7.3
 
       ANSI control sequences are stripped when writing to *file*.
 
-.. option:: -W
+   .. versionchanged:: 7.3
+      Add ``--write`` long option.
+
+.. option:: -W, --errors
```

Comment:
Would `--fail-fast` be a better name than `--errors`?
I don't find `--errors` particularly descriptive.

Reply:
Hmm, this option is often used in combination with `--keep-going`, my thinking was:

```
--errors --keep-going
```

And `--fail-fast` would seem to contradict it:

```
--fail-fast --keep-going
```

Open to other suggestions!

Reply:
If possible, I would have suggested `--fatal-warnings` because it's exactly what it does for `gcc`. Or we can go for `--warning-is-error` or `--warn-as-error` (but I would prefer the former).

Reply:
I prefer ``--fail-on-warning`` here, it's more descriptive, if a little wordier.

Reply:
I usually try to come with flags that are used by other well-known tools because it makes less options to remember overall, but I think yours is a bit betetr when used with `--keep-going`